core/iwasm/aot/aot_intrinsic.c

```diff
@@ -5,6 +5,8 @@
 
 #include "aot_intrinsic.h"
 
+#include <float.h>
```

Comment:
Should not include float.h here, normally we just include bh_platform.h, and implement the API in platform layer if necessary.

Reply:
Forget to remove this include